community-membership.md

```diff
@@ -26,7 +26,7 @@ below.
 --> 
 ## Member
 
-Members are continuously active contributors in the community. They can have issues and PRs assigned to them. Members are expected to participate in community discussions and remain active contributors to the community.  
+Members are continuously active contributors in the community. They can have issues and PRs assigned to them. Members are expected to participate in community discussions and remain active contributors to the community. Members can also run Radius functional tests without approval from Approvers or Maintainers.
```

Comment:
Members of the radius-project org (https://github.com/orgs/radius-project/people) is separate from the `members-radius` team (https://github.com/orgs/radius-project/teams/members-radius). I'd add a new section about this new team we have for running functional tests, or we create automation to sync the members of the org with the members-radius team if we want them to be the same.